Resources/Prototypes/Entities/Structures/Shuttles/thrusters.yml

```diff
@@ -64,9 +106,31 @@
       visible: false
       offset: 0, 1
 
+- type: entity
+  id: BadThruster
+  parent: [ BaseThrusterStatic, ConstructibleMachine ]
```

Comment:
BadThruster is a very generic and non-descriptive name.

You should just parent thruster + gyroscope and remove anchorable from it.

Reply:
I don't understand how I would remove anchorable if its inheriting it from BaseStructureDynamic through BaseThruster

Reply:
you will need to make a new parent without anchorable

Reply:
Isn't that what I did?

Reply:
Having slept, I realize you were asking for a combined thruster/gyro.

Reply:
It will require rewriting ThrusterSystem.cs since it uses switch case for checking the thruster type (Linear or Angular). Probably also would require some checks to make sure you can't double your thrust by spinning while moving.

Overall, I kinda dislike this solution, and think a better one would be to just have a wallmount gyro if space is the concern